spec/services/activity/updater_spec.rb

```diff
@@ -42,19 +42,16 @@
 
           before do
             updater.update(:commitment)
-            activity.build_commitment
           end
 
-          it "swallows the error and adds it to the activity's errors" do
-            expect(activity.errors.full_messages.first).to eq(
-              "Value Validation failed: Value Value is not a number"
+          it "adds the error to the commitment's errors" do
+            expect(activity.commitment.errors.full_messages.first).to include(
+              "Value is not a number"
             )
```

Comment:
Is this only part of the error message? (Or more specifically, why the shift to `include` vs `eq`?

Reply:
The model error generated for us by active record actually comes out as "`Value Value must be a valid number`" but I believe it's formatted in the view so it doesn't look like this. I thought writing that out felt a bit gross so changed this to `include` to assert on the thing the user sees. Do you think it'd be better to be explicit and assert on the exact error?

Reply:
I think it's better to make an assertion that the human readable stuff is there!